resolwe_bio/management/commands/generate_samples.py

```diff
@@ -0,0 +1,258 @@
+import gzip
+import io
+import json
+import os
+import random
+import shutil
+import string
+import zipfile
+
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from django.conf import settings
+from resolwe.flow.models import Data, DescriptorSchema, Process, Collection, Storage
+from resolwe_bio.tools.utils import escape_mongokey, gzopen
+
+
+class Command(BaseCommand):
+
+    """Generate test data"""
+
+    help = 'Generate test data'
```

Comment:
Put the string in `"` since this we try to follow the rule that we use double quotes for strings meant for human consumption and use single quotes for everything else.